code/game/machinery/bots/medbot.dm

```diff
@@ -46,10 +46,10 @@
 	name = "\improper Mysterious Medibot"
 	desc = "International Medibot of mystery."
 	skin = "bezerk"
-	treatment_oxy = "dexalinp"
-	treatment_brute = "bicaridine"
-	treatment_fire = "kelotane"
-	treatment_tox = "anti_toxin"
+	treatment_oxy = "salbutamol"
+	treatment_brute = "salglu_solution"
+	treatment_fire = "salglu_solution"
+	treatment_tox = "charcoal"
 

```

Comment:
just make this inject omnizine; this is a special medbot that is meant to be super OP.